### ISSUE TITLE
Are the pre-trained model shared trained by triplet loss or softmax loss?

### ISSUE THREAD

**Issue by Shawn-Shan** [davidsandberg/facenet#1130]
I see there are a couple shared models (i.e. 20180408-102900). I wonder whether they are trained by Triplet loss or softmax loss. 

From looking at the embeddings, it seems they are trained with softmax loss. If that is the case, is there any models trained by triplet loss that we can use? 

Thanks!

**Comment by Saqib-Mahmood** [comment]
I have same question. If there is any model that is trained with triplet loss?
Or how can we use Model that is trained with softmax loss in Face Recognition problem?

**Comment by seth814** [comment]
The pretrained models are both created from train_softmax.py
